### PR TITLE
Add readscale known issue

### PR DESCRIPTION
**Pull request by JamesFerebee**
IcM 462365331

# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

File: support/sql/releases/sqlserver-2019/cumulativeupdate24.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This article describes Cumulative Update package 24 (CU24) for Microsoft SQL Ser
 
 ## Known issues in this update
 
+###Issue 1:
```
Comment (MashaMSFT) [inline support/sql/releases/sqlserver-2019/cumulativeupdate24.md:26]:
```suggestion
### Issue 1:
```

File: support/sql/releases/sqlserver-2019/cumulativeupdate24.md [hunk 2]
```diff
@@ -31,6 +32,11 @@ SQL Server 2019 CU14 introduced a [fix to address wrong results in parallel plan
 
 Microsoft is working on a fix for this issue and it will be available in a future CU.
 
+###Issue 2:
```
Comment (JamesFerebee) [inline support/sql/releases/sqlserver-2019/cumulativeupdate24.md:35]: ### Read-scale availability group not displayed in dm_hadr_database_replica_cluster_states

File: support/sql/releases/sqlserver-2019/cumulativeupdate24.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ This article describes Cumulative Update package 24 (CU24) for Microsoft SQL Ser
 
 ## Known issues in this update
 
+### Access violation when session is reset
```
Comment (rwestMSFT) [inline support/sql/releases/sqlserver-2019/cumulativeupdate24.md:26]:
```suggestion
### Access violation when session is reset

```

File: support/sql/releases/sqlserver-2019/cumulativeupdate24.md [hunk 4]
```diff
@@ -31,6 +32,11 @@ SQL Server 2019 CU14 introduced a [fix to address wrong results in parallel plan
 
 Microsoft is working on a fix for this issue and it will be available in a future CU.
 
+### Read-scale availability group not displayed in dm_hadr_database_replica_cluster_states
```
Comment (rwestMSFT) [inline support/sql/releases/sqlserver-2019/cumulativeupdate24.md:35]:
```suggestion
### Read-scale availability group not displayed in dm_hadr_database_replica_cluster_states

```

File: support/sql/releases/sqlserver-2019/cumulativeupdate24.md [hunk 5]
```diff
@@ -31,6 +32,11 @@ SQL Server 2019 CU14 introduced a [fix to address wrong results in parallel plan
...
+SQL Server 2019 CU24 introduced fix 2714260 which causes an issue with sys.dm_hadr_database_replica_cluster_states for read-scale availability groups that results in the Availability Databases Folder in SQL Server Management Studio (SSMS) to not show the databases in the AG. To mitigate this issue, roll back the patch to CU23.
```
Comment (rwestMSFT) [inline support/sql/releases/sqlserver-2019/cumulativeupdate24.md:36]:
```suggestion
SQL Server 2019 CU24 introduced [fix 2714260](#2714260), which causes an issue with `sys.dm_hadr_database_replica_cluster_states` for read-scale availability groups that results in the **Availability Databases** folder in SQL Server Management Studio (SSMS) not showing the databases in the AG. To mitigate this issue, roll back the patch to CU23.
```

**Review by rwestMSFT — APPROVED** [review]

**Pull request closed without merging**